### PR TITLE
Critical update

### PR DESCRIPTION
**Pull request by KielD-01**
As per PHP 7.2 - `create_function()` is deprecated and anonymous function should be as the replacement. So, it was also done here.
The code has been reformatted and re-analyzed.

### CODE REVIEW

**Comment by KielD-01** [comment]
Dear @RaonyMarcondes, please, make a review of this pull request as soon as possible.
Also, will be great, if You will be able to make a release version, like 0.x

**Comment by KielD-01** [comment]
@eminetto , please, have a look at this one, if You have a chance.
Thanks in advance.

File: src/phpQuery.php [hunk 1]
```diff
@@ -18,1300 +18,1420 @@
...
-			print var_dump($text);
-	}
-	/**
-	 * Make an AJAX request.
-	 *
-	 * @param array See $options http://docs.jquery.com/Ajax/jQuery.ajax#toptions
-	 * Additional options are:
-	 * 'document' - document for global events, @see phpQuery::getDocumentID()
-	 * 'referer' - implemented
-	 * 'requested_with' - TODO; not implemented (X-Requested-With)
```
Comment (lucassabreu) [inline src/phpQuery.php]: @KielD-01 can you keep this?

File: src/phpQuery/phpQueryObject.php [hunk 2]
```diff
@@ -1,3180 +1,3584 @@
...
-			$this->elementsBackup = $this->elements;
-		$notSimpleSelector = array(' ', '>', '~', '+', '/');
-		if (! is_array($selectors))
-			$selectors = $this->parseSelector($selectors);
-		if (! $_skipHistory)
-			$this->debug(array("Filtering:", $selectors));
-		$finalStack = array();
-		foreach($selectors as $selector) {
-			$stack = array();
-			if (!
```
Comment (lucassabreu) [inline src/phpQuery/phpQueryObject.php]: can you revert this?

**Review by lucassabreu — COMMENTED** [review]
@KielD-01 thank you for the contribution and cleaning up the source..

there is some points I would like that you "reverted", where were some method descriptions

Once you do this I will merge the PR, and create a git tag/branch for it

File: src/phpQuery/phpQueryObject.php (hunk 2 above)
Comment (KielD-01) [reply src/phpQuery/phpQueryObject.php]: Yes @lucassabreu, if this is going to be maintained

File: src/phpQuery.php (hunk 1 above)
Comment (KielD-01) [reply src/phpQuery.php]: Sure

File: src/phpQuery/phpQueryObject.php (hunk 2 above)
Comment (lucassabreu) [reply src/phpQuery/phpQueryObject.php]: ow, it marked too much

Comment (lucassabreu) [reply src/phpQuery/phpQueryObject.php]: ignore this for now

Comment (KielD-01) [reply src/phpQuery/phpQueryObject.php]: Np

Comment (lucassabreu) [reply src/phpQuery/phpQueryObject.php]: well, there was a docblock here that i wanted to keep, but seens that was just some glitch on githubs diff, no changes needed

Comment (KielD-01) [reply src/phpQuery/phpQueryObject.php]: So, no reverting needed?

Comment (lucassabreu) [reply src/phpQuery/phpQueryObject.php]: no reverting needed

**Pull request merged**